README.md

```diff
@@ -78,10 +80,9 @@ The `:manager:assembleDebug` task generates a debuggable server. You can attach
 
 All code files in this project are licensed under Apache 2.0
 
-Under Apache 2.0 section 6, specifically:
-
-* You are **FORBIDDEN** to use `manager/src/main/res/mipmap*/ic_launcher*.png` image files, unless for displaying Shizuku itself.
+Under Apache 2.0 section 6, you are NOT granted permission to use the following common law trademarks
 
-* You are **FORBIDDEN** to distribute the apk that use Shizuku as name
-or use `moe.shizuku.privileged.api` as application id or declare `moe.shizuku.manager.permission.*` permission
-to any store (IBNLT Google Play Store, F-Droid, Amazon Appstore etc.).
+* The application icons, contained in the `manager/src/main/res/mipmap*/ic_launcher*.png` image files
+* The application name, Shizuku™
+* The application ID, `moe.shizuku.privileged.api`
+* The `moe.shizuku.manager.permission.*` Android permission
```

Comment:
I still am not a lawyer.

Note: the android permission would _likely_ be ruled to not be a common law trademark. It would need to be used prominently in the project's branding. It also violates OSI freedom 3, which would make this unfree software again even though it uses Apache-2.0. Are you sure you want to include this?